### PR TITLE
Info view

### PR DESCRIPTION
**Pull request by misternebula**

### CODE REVIEW

File: QSB/PlayerInfo.cs [hunk 1]
```diff
@@ -13,14 +13,43 @@ public class PlayerInfo
...
-        public string Name { get; set; }
+
+        private string _name;
+        public string Name
+        {
+            get
+            {
+                return _name;
+            }
+
```
Comment (amazingalek) [inline QSB/PlayerInfo.cs:24]: `get =>_name;`

File: QSB/PlayerInfo.cs [hunk 2]
```diff
@@ -13,14 +13,43 @@ public class PlayerInfo
...
         public bool IsReady { get; set; }
-        public Transform ReferenceSector { get; set; }
+        private Transform _referenceSector;
+        public Transform ReferenceSector
+        {
+            get
+            {
+                return _referenceSector;
+            }
+
```
Comment (amazingalek) [inline QSB/PlayerInfo.cs:40]: `get =>_referenceSector;`

File: QSB/TransformSync/PlayerTransformSync.cs [hunk 3]
```diff
@@ -51,13 +52,18 @@ protected override Transform InitRemoteTransform()
             GetComponent<AnimationSync>().InitRemote(body);
 
             var marker = body.gameObject.AddComponent<PlayerHUDMarker>();
-            marker.SetId(netId.Value);
+            marker.SetId(GetAttachedNetId());
```
Comment (amazingalek) [inline QSB/TransformSync/PlayerTransformSync.cs:55]: oh, was it wrong before?

**Comment by amazingalek** [comment]
image
Who's Jeff & Jeff?

File: QSB/TransformSync/PlayerTransformSync.cs (hunk 3 above)
Comment (misternebula) [reply QSB/TransformSync/PlayerTransformSync.cs:55]: they mean the same thing but it's more consistent with other files

**Comment by misternebula** [comment]
> image
> Who's Jeff & Jeff?

I'd just finished playing half life alyx when i made this so a certain chapter was still on my mind :P I probably should have just made them blank

**Comment by amazingalek** [comment]
> I'd just finished playing half life alyx when i made this so a certain chapter was still on my mind :P I probably should have just made them blank

Ok, but why are there 3 things with the same net id?

**Pull request closed without merging**